test/unit/search-bar-test.spec.js

```diff
@@ -41,13 +41,6 @@ describe('Test Search Bar : ', () => {
       done();
     }).catch( e => { console.log(e.toString()) });
   });
-  it('Expect search button icon to be "search"', (done) => {
```

Comment:
@Rodriguez-Carlos-BAH this unit test was removed, do we need to add it back?

Reply:
The unit test is no longer valid.